Sources/Mail.php

```diff
@@ -14,6 +14,8 @@
 namespace SMF;
 
 use SMF\Db\DatabaseApi as Db;
+use SMF\Search\SearchApi;
```

Comment:
Why?

Reply:
I didn't see it, but guessing it was added automatically by the IDE